### ISSUE TITLE
opening an abstract in a new tab results in empty page

### ISSUE THREAD

**Issue by romanchyla** [adsabs/bumblebee#1630]
### Actual Behavior

![image](https://user-images.githubusercontent.com/411794/48230520-50be5900-e379-11e8-8be9-9114add869e1.png)

### Steps to Reproduce

search for: https://ui.adsabs.harvard.edu/#search/p_=0&q=author%3A%22diaconis%22&sort=date%20desc%2C%20bibcode%20desc

middle click on:
https://ui.adsabs.harvard.edu/#abs/2018arXiv180207122D/abstract

results in empty page:

see screenshot

the only request that seems to be failing was:

curl 'https://ui.adsabs.harvard.edu/v1/graphics/2018arXiv181000409B' -H 'dnt: 1' -H 'accept-encoding: gzip, deflate, br' -H 'accept-language: en-US,en;q=0.9,cs-CZ;q=0.8,cs;q=0.7,fr-FR;q=0.6,fr;q=0.5' -H 'authorization: Bearer:xxxxxxxxxxxx' -H 'cookie: NASA_ADS_TMP=; gsScrollPos-5817=318; gsScrollPos-845=0; gsScrollPos-849=0; gsScrollPos-865=; gsScrollPos-853=; _ga=GA1.2.2004965332.1458073021; gsScrollPos-3368=0; __unam=fb897b-162b0ca1da1-4b4e88ae-2; route=53414b266a0b40fe0225206655aa25c847fd6870; sroute=f5095ea9f58e211567c8093cf24900db742f48f6; gsScrollPos-5062=0; gsScrollPos-3238=0; _gid=GA1.2.405487565.1541712272; _gat=1; session=.eJyrVopPK0otzlCyKikqTdVRis9MUbKqVlJIUrJS8nNxyvLNSjaIyg0t9831NfHLysjyD_c19g8PLY9y8TX2dY-s8A_JyPSrCrVVqgXqLUgtyk3MS80rgZlWWpxaBDZRycRMqRYAgdMjXw.W-SzAg.E-eroR42ztswg6GbpXyiZA5fcJ0' -H 'pragma: no-cache' -H 'x-requested-with: XMLHttpRequest' -H 'user-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36' -H 'content-type: application/x-www-form-urlencoded' -H 'accept: application/json, text/javascript, */*; q=0.01' -H 'cache-control: no-cache' -H 'authority: ui.adsabs.harvard.edu' -H 'referer: https://ui.adsabs.harvard.edu/' --compressed

response:

```{"Error Info": "No database entry found for 2018arXiv181000409B", "Error": "Unable to get results!"}
```

and 

```curl 'https://ui.adsabs.harvard.edu/v1/resolver/2018arXiv181000409B/associated' -H 'dnt: 1' -H 'accept-encoding: gzip, deflate, br' -H 'accept-language: en-US,en;q=0.9,cs-CZ;q=0.8,cs;q=0.7,fr-FR;q=0.6,fr;q=0.5' -H 'authorization: Bearer:xxxxx' -H 'cookie: NASA_ADS_TMP=; gsScrollPos-5817=318; gsScrollPos-845=0; gsScrollPos-849=0; gsScrollPos-865=; gsScrollPos-853=; _ga=GA1.2.2004965332.1458073021; gsScrollPos-3368=0; __unam=fb897b-162b0ca1da1-4b4e88ae-2; route=53414b266a0b40fe0225206655aa25c847fd6870; sroute=f5095ea9f58e211567c8093cf24900db742f48f6; gsScrollPos-5062=0; gsScrollPos-3238=0; _gid=GA1.2.405487565.1541712272; _gat=1; session=.eJyrVopPK0otzlCyKikqTdVRis9MUbKqVlJIUrJS8nNxyvLNSjaIyg0t9831NfHLysjyD_c19g8PLY9y8TX2dY-s8A_JyPSrCrVVqgXqLUgtyk3MS80rgZlWWpxaBDZRycRMqRYAgdMjXw.W-SzAg.E-eroR42ztswg6GbpXyiZA5fcJ0' -H 'pragma: no-cache' -H 'x-requested-with: XMLHttpRequest' -H 'user-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36' -H 'content-type: application/x-www-form-urlencoded' -H 'accept: application/json, text/javascript, */*; q=0.01' -H 'cache-control: no-cache' -H 'authority: ui.adsabs.harvard.edu' -H 'referer: https://ui.adsabs.harvard.edu/' --compressed```

**Comment by romanchyla** [comment]
note: the paper will load, after a while, so it seems to be related to some unhandled UI error that stops the widget flow (?)

**Comment by thostetler** [comment]
Yeah I'm thinking you're right -- this is the interim page (before any content is rendered).  I'm guessing it's waiting on entry to load, when you stay in the same window it reuses the content from the already downloaded results.

I haven't had it stop loading, like you -- but I can see a flash of empty content for a second or so before the abstract loads up.
